pkg/test/cmd/extension/command.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2021 Tetrate
```

Comment:
but I realized that this test utils don't have any `extension` specific stuff so we could move this to `pkg/test/cmd/command.go`. Even we have `NewRootCommand` which literally returns the root command of getenvoy which is not extension specific :smile: